subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -1564,7 +1564,7 @@
             <sample id="manifest" dir="userguide/tutorial/manifest" title="Customization of MANIFEST.MF">
                 <sourcefile file="build.gradle" snippet="add-to-manifest"/>
             </sample>
-            <para>You can create stand alone instances of a <classname>Manifest</classname>. You can use that for example,
+            <para>You can create standalone instances of a <classname>Manifest</classname>. You can use that for example,
```

Comment:
The word "stand-alone" should actually contain a dash. http://www.dictionary.com/browse/standalone. Would you mind fix that?